docs/user-guide/utils.rst

```diff
@@ -4,15 +4,83 @@
 Utilities
 *********
 
-Overview
-========
 This package provides a number of utilities to work with CCSDS packets.
 
+.. contents::
+   :depth: 2
+
+Iterating through Packet Bytes
+==============================
+During debugging it is often useful to break a stream of multiple packets into a list of byte sequences associated with each packet. That is, the i'th element of the list will be a `bytes` object assosciated with the i'th packet in the file.
+
+This can be done with the `utils.split_packet_bytes()` (returns a list) and `utils.iter_packet_bytes()` (returns a generator) functions. These functions have an optional keyword arugment `include_primary_header=True` which determines whether the primary header is included in the byte sequence returned. By default, it is included.
+
+This function works with mixed files containing multiple APIDs, which may include both fixed length and variable length packets.
+
+.. code-block:: python
+
+  from ccsdspy.utils import split_packet_bytes
+
+  packet_bytes = split_packet_bytes('mixed_file.bin')
+
+  # Print bytes of first five packets
+  for i in range(5):
+    print(f"The bytes in packet number {i+1} out "
+          f"of {len(packet_bytes)} are: ")
+    print(packet_bytes[i].hex())
+
+  print()
+  print("Stopping after first five!")
+
+
+This generates the following output for a sequence of variable length packets:
+
+.. code-block::
+
+  The bytes in packet number 1 out of 10 are:
```

Comment:
This statement is confusing. Sounds like you are going to print out 10 packets and this is the first. Would it be better to say "The 10 bytes in packet number 1 are:"

Reply:
I removed these print statements